spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe "contestants index page" do
+    it "render an index page with all contestants names" do
+        # User Story 2 of 3
+        # challenge = Challenge.create!(theme: "Apartment Furnishings")
+        # project_1 = Project.create!(name: "Litfit", material: "Lamp Shade", challenge_id: 3)
+        # project_2 = Project.create!(name: "Rug Tuxedo", material: "string", challenge_id: 3)
+        # project_3 = Project.create!(name: "LeatherFeather", material: "Lether", challenge_id: 4)
+        contestant_1 = Contestant.create!(name: "Kentaro Kameyama", age: 5, hometown: "Las Vagas", years_of_experience: 10)
+        contestant_2 = Contestant.create!(name: "Jay McCarroll", age: 5, hometown: "New York City", years_of_experience: 3)
+        # As a visitor,
+        # When I visit the contestants index page ("/contestants")
+        visit "/contestants"
+        # I see a list of names of all the contestants
+        expect(page).to have_content(contestant_1.name)
+        expect(page).to have_content(contestant_2.name)
+        # And under each contestants name I see a list of the projects (names) that they've been on
+        expect(page).to have_content(contestant_1.project)
+        expect(page).to have_content(contestant_2.project)
+        
```

Comment:
There are a few things with the setup of this test that probably made it difficult to test. To set this up properly, you would need to create a challenge, associate projects to that challenge, and then contestants to projects.

```ruby
# When creating a challenge, don't forget the budget.
challenge = Challenge.create!(theme: "Apartment Furnishings", project_budget: 1000)

project_1 = Project.create!(name: "Litfit", material: "Lamp Shade", challenge_id: challenge.id)
project_2 = Project.create!(name: "Rug Tuxedo", material: "string", challenge_id: challenge.id)
project_3 = Project.create!(name: "LeatherFeather", material: "Lether", challenge_id: challenge.id)

contestant_1 = Contestant.create!(name: "Kentaro Kameyama", age: 5, hometown: "Las Vagas", years_of_experience: 10)
contestant_2 = Contestant.create!(name: "Jay McCarroll", age: 5, hometown: "New York City", years_of_experience: 3)

ContestantProject.create!(contestant_id: contestant_1.id, project_id: project_1.id)
ContestantProject.create!(contestant_id: contestant_1.id, project_id: project_2.id)
ContestantProject.create!(contestant_id: contestant_2.id, project_id: project_3.id)
```

With this you would be able to now get all the projects for a contestant.